lms/templates/edxnotes/toggle_notes.html

```diff
@@ -1,7 +1,9 @@
+<%page expression_filter="h"/>
 <%!
 import json
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
+from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
 %>
 <%namespace name='static' file='/static_content.html'/>
 <%page args="course"/>
```

Comment:
The linter should have given an error because you have two page expressions. These args must move up to the top.  Let me know of the linter isn't working for you for some reason.